packages/office-addin-debugging/README.md

```diff
@@ -98,9 +103,9 @@ Syntax:
 
 Options:
 
-`--unload <command>`
+` --prod`
+
+Specifies that debugging session is for production mode. Default is dev mode.
```

Comment:
"Default is **development** mode."  Although does this actually get used?

Reply:
we need to be able to contruct the manifest for SDX's so we would need to know the mode (prod or dev)

Reply:
I need to rethink how stop is going to work

Reply:
@TCourtneyOwen Can you change "dev mode" to "development mode" in the help text?